README.md

```diff
@@ -166,7 +183,7 @@ The aforementioned instructions rely on [Autotools](https://www.gnu.org/software
 1. `cmake -S. -Bbuild`
 2. `cd build`
 3. ```make -j 4```
-4. ```ctest -j 4```
+4. ```ctest -j $(nproc)```
```

Comment:
I wouldn't use `$(nproc)`. People will just copy/paste and it will not work. I suggest that for the tests we don't use `-j` because I think that the Fortran test can fail when run in parallel (not sure).